addons/account_peppol/views/res_config_settings_views.xml

```diff
@@ -7,162 +7,109 @@
-                                       readonly_items="account_peppol_mode_constraint != 'prod' and ['prod'] or []"
-                                       hidden_items="account_peppol_mode_constraint != 'test' and ['test'] or []"
-                                       options="{'horizontal': true }"/>
-                                    <div class="mb-3" invisible="not account_peppol_edi_mode == 'prod'">
-                                        By clicking the button below I accept that Odoo may process my e-invoices.
-                                    </div>
-                                    <div class="mb-3" invisible="not account_peppol_edi_mode == 'test'">
-                                        Test mode allows registration of the user on the test Peppol network.
-                                        By clicking the button below I accept that Odoo may process my e-invoices.
-                                    </div>
-                                    <div class="mb-3" invisible="not account_peppol_edi_mode == 'demo'">
-                                        In demo mode sending and receiving invoices is simulated. There will be no communication with the Peppol network.
-                                    </div>
-                                </div>
-                                <div class="d-flex gap-1 action_buttons" colspan="3">
-                                    <widget name="peppol_settings_buttons"
-                                            invisible="account_peppol_proxy_state not in ('not_registered', 'not_verified', 'sent_verification', 'pending', 'manually_approved', 'active')"/>
-                                    <div class="mt-3"
-                                         invisible="account_peppol_proxy_state in ('not_registered', 'active', 'rejected', 'canceled')">
-                                        <button name="button_cancel_peppol_registration"
-                                                type="object"
-                                                string="Cancel registration"
-                                                class="btn btn-secondary"/>
-                                    </div>
+                            </div>
+
+                            <div class="mt-4"
+                                 invisible="account_peppol_proxy_state not in ('sender', 'smp_registration', 'receiver', 'rejected')">
+                                Peppol participant status:
+                                <b>
+                                    <field name="account_peppol_proxy_state"
+                                           class="oe_inline"
+                                           readonly="1"
+                                           decoration-danger="account_peppol_proxy_state == 'rejected'"
+                                           decoration-info="account_peppol_proxy_state in ('sender', 'smp_registration', 'receiver')"/>
```

Comment:
decoration-info gives a weird look and feel: we have the feeling we can click on it but it's not clickable :see_no_evil: 

Remove the `<b/>`

Can we make the field's label looking more like a label in odoo? ... like using `o_form_label` maybe?